admin-src/DiagnoseResponse/index.js

```diff
@@ -96,39 +105,63 @@ export const DiagnoseResponse = ({ query }) => {
         .firestore()
         .doc(`users/${currentDiagnose.userUID}/requests/${currentDiagnose.id}`)
         .update({
-          updatedAt: firebase.firestore.FieldValue.serverTimestamp()
+          updatedAt: firebase.firestore.FieldValue.serverTimestamp(),
+        })
+    }
+
+    const setAdminIsLastAnswer = async () => {
+      await firebase
+        .firestore()
+        .doc(`users/${currentDiagnose.userUID}/requests/${currentDiagnose.id}`)
+        .update({
+          adminIsLastAnswer: true,
         })
     }
 
     await writeNewResponse()
     await changeUpdateAtFieldForRequest()
+    await setAdminIsLastAnswer()
 
-    setDiagnoses(diagnoses.filter(diagnose => diagnose.id !== currentDiagnose.id))
+    setDiagnoses(
+      diagnoses.filter(diagnose => diagnose.id !== currentDiagnose.id),
+    )
     setCurrentDiagnose(null)
     setIsSubmitting(false)
   }
 
-  const fetchDiagnoseImagesAndSetAsCurrent = async (diagnose) => {
+  const fetchDiagnoseImagesAndSetAsCurrent = async diagnose => {
     setCurrentDiagnose(null)
     const images = await getDownloadURLFromImages(diagnose.imageReferences)
     setCurrentDiagnose({
       ...diagnose,
-      images
+      images,
     })
   }
 
   return (
     <div className={classes.row}>
       <ListGroup className={classes.column}>
-        {diagnoses && diagnoses.map((diagnose, index) => (
-          <ListGroup.Item action key={index} onClick={() => fetchDiagnoseImagesAndSetAsCurrent(diagnoses[index])}>
-            {diagnose.id}{' '} - {diagnose.username} ({firebaseTimestampToMoment(diagnose.createdAt).format('LL')})
-          </ListGroup.Item>
-        ))}
+        {diagnoses &&
+          diagnoses.map((diagnose, index) => (
+            <ListGroup.Item
+              action
+              key={index}
+              onClick={() =>
+                fetchDiagnoseImagesAndSetAsCurrent(diagnoses[index])
+              }>
```

Comment:
`>` should be the only token in the line. Move `}` to `fetchDiagnoseImagesAndSetAsCurrent(diagnoses[index])`

(StandardJS complains)